src/db/setups/users.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const Users = require('../models/users');
```

Comment:
Consider renaming the imported module to `User` (singular) instead of `Users` to better reflect that it represents a single user model. 🤔